src/status_im/contexts/settings/wallet/saved_addresses/events.cljs

```diff
@@ -0,0 +1,95 @@
+(ns status-im.contexts.settings.wallet.saved-addresses.events
+  (:require
+    [status-im.constants :as constants]
+    [status-im.contexts.wallet.data-store :as data-store]
+    [taoensso.timbre :as log]
+    [utils.re-frame :as rf]))
+
+(defn save-address
+  [{:keys [db]}
+   [{:keys [address name customization-color on-success on-error chain-short-names ens]
+     :or   {on-success          (fn [])
+            on-error            (fn [])
+            name                ""
+            customization-color constants/account-default-customization-color
+            ens                 ""
+            ;; the chain short names should be a string like eth: or eth:arb:oeth:
+            chain-short-names   (str constants/mainnet-short-name ":")}}]]
```

Comment:
There is no harm in removing it. Since we have not started with add/edit save address flow (this event is not used yet), I left the defaults if someone wants to try it from REPL, in the meantime. But, I clean this up we do the add/edit save address flow 👍